test/objects.test.ts

```diff
@@ -95,6 +95,20 @@ describe("Objects", () => {
     });
   });
 
+  it("PartialDeep", () => {
+    type res0 = Call<Objects.PartialDeep, { a: 1; b: 2 }>;
+    //    ^?
+    type test0 = Expect<Equal<res0, { a?: 1; b?: 2 }>>;
+
+    type res1 = Call<Objects.PartialDeep, { a: 1; b: { c: 2 } }>;
+    //    ^?
+    type test1 = Expect<Equal<res1, { a?: 1; b?: { c?: 2 } }>>;
+
+    type res2 = Call<Objects.PartialDeep, { a: 1; b: { c: 2; d: { e: 3 } } }>;
+    //    ^?
+    type test2 = Expect<Equal<res2, { a?: 1; b?: { c?: 2; d?: { e?: 3 } } }>>;
```

Comment:
can you add tests with :
- an array of objects
- an object with properties being array of abjects

Reply:
done :)

![image](https://user-images.githubusercontent.com/44379716/221383213-0583d65d-4300-464d-9d1f-8519b6bb6ff3.png)

Reply:
Oh sorry i meant a tuple. Really sorry for confusion

Reply:
No problem, it's done!